src/garage/np/exploration_strategies/gaussian_strategy.py

```diff
@@ -25,8 +23,43 @@ def __init__(self,
         self._action_space = env_spec.action_space
 
     def get_action(self, t, observation, policy, **kwargs):
+        """
+        Get action from this policy for the input observation.
+
+        Args:
+            t: Iteration.
```

Comment:
please make sure all args and returns have types.

Reply:
Yeah, at a glance it's not obvious what type t is (it could also benefit from a longer name, like `iteration`).